src/behavior/features/policyManagement.feature

```diff
@@ -9,7 +9,8 @@ Feature: Create a new policy
     And I can retrieve the created policy for tenant "511" from the Access Control
 
   Scenario Outline: Request validation
-    Given I send a policy creation request to the Access Control for tenant "<tenant>" and subject "<subject>"
+    Given I send a remove request for tenant "511"
```

Comment:
This seems to be a initialization condition in the tests. Is it common to all the tests? There is a @before.each_scenario for that in the terrain.py file. Why don't you use that?